### DISCUSSION TITLE
grafana mivus2.0 没有数据

### DISCUSSION THREAD

**Discussion by fangshirui** [milvus-io/milvus#17156] [Q&A and General discussion] [answered]
按照官方文档配置好后， 导入 mivus2.0 面板，发现没有数据。 并报了一些错误，这个是什么情况。 
用的是 helm 安装的 milvus-3.0.23  appVersion  2.0.2。

grafana 中其他一些服务器信息是有的。  通过 kubectl get servicemonitor 命令看 servicemonitor 也是服务启动了的。 

<img width="1970" alt="image" src="https://user-images.githubusercontent.com/45707105/169740991-36e8efdc-df31-4e77-ac37-5a1af68d5435.png">

**Comment by LoveEachDay** [comment, marked as answer]
Could you provide the grafana version? You'd better use grafana 8.x.

**Reply by fangshirui** [reply]
grafana version 8.3.3  from official doc .  `git clone https://github.com/prometheus-operator/kube-prometheus.git`

**Reply by fangshirui** [reply]
<img width="743" alt="image" src="https://user-images.githubusercontent.com/45707105/169742090-9fc344e8-2a77-4567-b444-603d2992af34.png">
这里可否看到哪里不对呢，，

**Reply by fangshirui** [reply]
doc is 8.3.3 but actually it's use 8.1.1 。  😖

**Reply by LoveEachDay** [reply]
> <img alt="image" width="743" src="https://user-images.githubusercontent.com/45707105/169742090-9fc344e8-2a77-4567-b444-603d2992af34.png">
> 
> 这里可否看到哪里不对呢，，

Seems the servicemonitor is ok. 
We've upgrade the dashboard for milvus 2.1.0, If you deploy a milvus cluster with 2.0.2, you'd better use this dashboard: https://github.com/milvus-io/milvus/blob/2.0.2/deployments/monitor/grafana/milvus-dashboard.json

**Reply by fangshirui** [reply]
<img width="1971" alt="image" src="https://user-images.githubusercontent.com/45707105/169784642-a6bce59a-ef56-4c9a-9205-a1b8728183ab.png">
use json for 2.0.2。 just look like this.  There are still many empty Windows.  Is this normal?

**Reply by LoveEachDay** [reply]
Yes, we've refactoring the metrics all along since milvus 2.0. And we'll release the full version in milvus 2.1.0.

**Reply by dushulin** [reply]
I alse meet the same question, we use 8.x grafana, and milvus cluster by helm, prometheus operator
we check the thenos data is correct, we can see several metrics as below, but we use 2.3x grafana json file import to grafana dashborad, we can't see anything!
<img width="1785" alt="image" src="https://github.com/milvus-io/milvus/assets/25276301/a319d538-b997-4696-92af-5d3198302169">
<img width="249" alt="image" src="https://github.com/milvus-io/milvus/assets/25276301/6a9223a0-0791-419f-8b85-80881c088051">

**Reply by dushulin** [reply]
<img width="1708" alt="image" src="https://github.com/milvus-io/milvus/assets/25276301/2217df0e-a4bb-4590-b491-2c45a5921dbd">

**Reply by Zmahl** [reply]
I am getting this same issue currently as well
